src/main/java/seedu/address/commons/util/LoanSlipDocument.java

```diff
@@ -0,0 +1,218 @@
+package seedu.address.commons.util;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.util.Arrays;
+
+import com.itextpdf.io.font.PdfEncodings;
+import com.itextpdf.io.image.ImageData;
+import com.itextpdf.io.image.ImageDataFactory;
+import com.itextpdf.kernel.font.PdfFont;
+import com.itextpdf.kernel.font.PdfFontFactory;
+import com.itextpdf.layout.Document;
+import com.itextpdf.layout.borders.Border;
+import com.itextpdf.layout.element.Cell;
+import com.itextpdf.layout.element.Image;
+import com.itextpdf.layout.element.Paragraph;
+import com.itextpdf.layout.element.Table;
+import com.itextpdf.layout.property.HorizontalAlignment;
+import com.itextpdf.layout.property.TextAlignment;
+
+/**
+ * Instance class to handle a single pdf document.
+ */
+public class LoanSlipDocument {
+
+    private static final int HEADER_FONT_SIZE = 32;
+    private static final int SUB_HEADER_FONT_SIZE = 22;
+    private static final int MID_HEADER_FONT_SIZE = 28;
+    private static final int PARAGRAPH_FONT_SIZE = 20;
+
+    private static final String FONT = "src/main/resources/font/Lato-Black.ttf";
+    private static final String LOGO_PATH = "src/main/resources/images/LiBerryLogo.png";
+    private static final String LINE_DIVIDER = "_______________________________________________________";
+
+    private Document doc;
+    private Table table;
+
+    /**
+     * Constructor for LoanSlipDocument object.
+     *
+     * @param doc iText Document to be wrapped.
+     */
+    public LoanSlipDocument(Document doc, Table table) {
+        this.doc = doc;
+        this.table = table;
+    }
+
+    /**
+     * Appends the logo to the pdf document.
+     */
+    public void writeLogo() {
+        try {
+            Image pdfImg = createImage(LOGO_PATH);
+            double newWidth = pdfImg.getImageWidth() * 0.1;
+            double newHeight = pdfImg.getImageHeight() * 0.1;
```

Comment:
Should the 0.1 be extracted to a final variable too?

Reply:
Good suggestion!